src/ResourceManager/ApplicationInsights/Commands.ApplicationInsights/packages.config

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.3" targetFramework="net45" />
```

Comment:
@frankguodongchen similarly to the .csproj file using `Common.Dependencies.targets`, you can remove a lot of the packages in this file because they should be referenced in the [`packages.config`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ResourceManager.Common/packages.config) file for `Commands.ResourceManager.Common`